beacon_node/operation_pool/src/attestation.rs

```diff
@@ -99,8 +98,11 @@ impl<'a, T: EthSpec> AttMaxCover<'a, T> {
 
                 let mut proposer_reward_numerator = 0;
 
+                // FIXME(sproul): store base_reward in reward cache
```

Comment:
I think we can resolve this by using the base reward from the `EpochCache`. Trying that now